src/keboola_mcp_server/tools/components/tools.py

```diff
@@ -88,10 +79,7 @@ async def retrieve_components_configurations(
         Sequence[str],
         Field(description='List of component IDs to retrieve configurations for. If none, return all components.'),
     ] = tuple(),
-) -> Annotated[
-    list[ComponentWithConfigurations],
-    Field(description='List of objects, each containing a component and its associated configurations.'),
-]:
+) -> RetrieveComponentsConfigurationsOutput:
```

Comment:
so we're not doing the `Annotated` any more? I'm not against it, but have discussed it?  Is it because it doesn't provide any useful info to the agent?